packages/database/src/modelClasses/Survey.js

```diff
@@ -15,4 +15,8 @@ export class SurveyModel extends DatabaseModel {
   get DatabaseTypeClass() {
     return SurveyType;
   }
+
+  async getSurveysBySurveyGroupId(surveyGroupId) {
+    return this.find({ survey_group_id: surveyGroupId });
+  }
```

Comment:
To me this is simple enough to do inline every time, rather than treat as a util

Reply:
It also doesn't seem to be used?